riyaz/schema.sql

```diff
@@ -64,6 +64,24 @@ create table course_outline (
     orphan boolean default('f')
 );
 
+create table store (
+    id integer primary key,
+    key unique,
```

Comment:
Shouldn't it be `key text unique`?

Reply:
Oh yeah!
This change was also supposed to be in the last commit. I must've forgotten to add it then.

Oddly though, live reload was working fine for me despite this.